### ISSUE TITLE
Are you thinking about some future work?

### ISSUE THREAD

**Issue by ravpacheco** [takenet/elephant#1]
Great job! How someone can help you ? Are you planing some feature ?
Maybe support for MongoDb, what you think ?

**Comment by andrebires** [comment]
Yes, please, be my guest. Its easier now since the official Mongo driver (finally) supports async IO.
https://github.com/mongodb/mongo-csharp-driver/
